corehq/apps/app_manager/static/app_manager/js/summary/models.js

```diff
@@ -7,9 +7,10 @@ hqDefine('app_manager/js/summary/models',[
     'knockout',
     'underscore',
     'app_manager/js/summary/utils',
+    'hqwebapp/js/initial_page_data',
     'hqwebapp/js/assert_properties',
     'hqwebapp/js/layout',
-], function ($, ko, _, utils, assertProperties, hqLayout) {
+], function ($, ko, _, utils, initialPageData, assertProperties, hqLayout) {
```

Comment:
Most of our knockout models don't use initial page data directly, they get their initial data, including urls, passed in as options. That said, I can't perfectly articulate *why* we have this convention. I do think it makes models easier to test. Thoughts?

Reply:
It's a good question. In this particular instance, I didn't actually make this decision as I was just moving code around. In general though I kind of prefer the model to access the initial page data directly. It keeps everything close at hand, and reduced the number of required options for each model, which feels more simple and encapsulated.